src/lib/gundb/UserStorageClass.js

```diff
@@ -1165,11 +1170,16 @@ export class UserStorage {
           avatar = `${process.env.PUBLIC_URL}/favicon-96x96.png`
         }
         break
+
+      case 'bonus':
```

Comment:
please, use constant

Reply:
@StanislavShevchenko see new commit with the changes.